README.md

```diff
@@ -35,35 +36,36 @@ It is intended that connecting the bits together into an engine be done at the l
 
 ### Advantages of Libdogecoin
 
-* No dependencies in case no p2p network client is required (only dependency is [libsecp256k1](https://github.com/bitcoin-core/secp256k1) added as git subtree)
-* The only dependency for the p2p network client is [libevent](https://github.com/libevent/libevent) (very portable)
-* optimized for MCU and low mem environments
-* ~full test coverage
-* mem leak free (valgrind check during CI)
+- No dependencies in case no p2p network client is required (only dependency is [libsecp256k1](https://github.com/bitcoin-core/secp256k1) added as git subtree)
+- The only dependency for the p2p network client is [libevent](https://github.com/libevent/libevent) (very portable)
+- optimized for MCU and low mem environments
+- ~full test coverage
+- mem leak free (valgrind check during CI)
```

Comment:
We should say "best effort to be memory leak free" -- valgrind does not catch all cases and in our case **has not**, as we've had memory addressing situations that MSVC's debugger caught that valgrind didn't see.  So I don't think we should make that as a statement; we should state it as a priority effort. (Which we do. But again valgrind is not perfect; and we should probably add a few more tools to our stable on that -- and in my opinion, just an opinion: occasional integrative smoke tests performed by humans as "spot checks", not just CI.)